services/dynamic-sidecar/tests/unit/test_api_containers.py

```diff
@@ -44,7 +120,7 @@ def client(
     ensure_external_volumes: tuple[DockerVolume],
     cleanup_containers,
     test_client: TestClient,
-):
+) -> TestClient:
```

Comment:
question: why not just overriding the fixture instead of creating yet another one? or at least give it a clear name why it's different cause we have so many...

Reply:
will go for a rename